lib/graphql/schema/printer.rb

```diff
@@ -0,0 +1,131 @@
+module GraphQL
+  # Used to convert your {GraphQL::Schema} to a GraphQL schema string
+  #
+  # @example print your schema to standard output
+  #   Schema = GraphQL::Schema.new(query: QueryType)
+  #   puts GraphQL::Schema::Printer.print_schema(Schema)
+  #
+  module Schema::Printer
+    extend self
```

Comment:
Cool, that sounds good to me. Yeah, it's nice that those methods are all stateless!